sdk/src/types/block/output/mod.rs

```diff
@@ -79,20 +79,20 @@ pub enum OutputBuilderAmount {
 }
 
 /// Contains the generic [`Output`] with associated [`OutputIdProof`] and [`OutputMetadata`].
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Eq, PartialEq)]
 #[cfg_attr(
     feature = "serde",
     derive(serde::Serialize, serde::Deserialize),
     serde(rename_all = "camelCase")
 )]
-pub struct OutputWithMetadata {
+pub struct OutputWithMetadataResponse {
```

Comment:
Move it with the responses?

Reply:
Since we went this route I think there's nothing that prevents us now from doing that. Give it a go, and I'll make the change.

Reply:
No sure what you mean by "give it a go"

Reply:
Give your "go" I meant lol

Reply:
Ahhh, yeah I do give me go then, was a rhetoric question :trollface: